javascript/javascript2/week1/wordsandletters.js

```diff
@@ -0,0 +1,56 @@
+/// Find the shortest word
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+function findShortestWord(list) {
+  let shortestWord = list[0];
+  for (let i = 0; i < list.length; i++) {
+    if (shortestWord.length > list[i].length) {
+      shortestWord = list[i];
+    } else {
+      continue;
+    }
+  }
+  return shortestWord;
+}
+
+console.log(findShortestWord(danishWords)); /// output ø
+
+//Find and count the Danish letters
+const danishString = "Jeg har en blå bil";
+const danishString2 = "Blå grød med røde bær";
+
+function findDanishLetters(letters) {
+  const danishLetters = {
+    total: 0,
+    å: 0,
+    æ: 0,
+    ø: 0,
+  };
+
+  for (let i = 0; i < letters.length; i++) {
+    switch (letters[i]) {
```

Comment:
Nice solution, however I would skip the erase letters with 0 and instead I would start with an empty object and just add a letter only if it appears. You could do something like ` danishLetters.æ = (danishLetters.æ || 0) + 1`, for example.